openprescribing/measure_definitions/lpzomnibus.json

```diff
@@ -23,6 +23,7 @@
   "tags_focus": [
     "lowpriority"
   ],
+  "include_in_alerts": false,
```

Comment:
From our conversation earlier, I'd understood the decision was to include the omnibus but not the individual LP measures?

Reply:
Oh I'm sorry I think I misunderstood:

> Fair enough, but I now think it'd be useful for individual alerts to start to go out, as we're some time down the line on the programme.

I guess your code does this!